lib/plugins/inspec-suggest/lib/inspec-suggest/suggtool/bump.rb

```diff
@@ -0,0 +1,66 @@
+require_relative "base"
+
+module InspecPlugins::Suggest
+  class SuggToolCommand < Inspec.plugin(2, :cli_command)
+
+    option :patch, type: :boolean, default: true,
+      desc: "Increase third digit of the version number"
+    option :minor, type: :boolean, default: false,
+      desc: "Increase second digit of the version number"
+    option :major, type: :boolean, default: false,
+      desc: "Increase first digit of the version number"
+
+    desc "bump SETNAME", "Increase version number of SETNAME in inspec.yml."
+    def bump(*requested_sets)
+      setup(requested_sets)
+
+      with_requested_sets do |set_name, set_cfg|
+        with_matching_yaml_files(set_cfg) do |control_file_path|
+          # TODO: We may not need the full path here.
+          file_info = {
+            full_path: control_file_path,
+            rel_path: control_file_path.sub(Inspec.src_root + "/", ""),
+          }
+          bump_version(file_info[:rel_path])
```

Comment:
This calling pattern is clearly copied from `inspec suggtool check` - it doesn't make sense here. The only thing you need is the path.